packages/react/src/components/Tooltip/index.tsx

```diff
@@ -130,21 +141,26 @@ export default function Tooltip({
         event.key === 'Esc' ||
         event.keyCode === 27
       ) {
+        event.preventDefault();
+        event.stopPropagation();
```

Comment:
Generally speaking, `stopPropagation()` is an anti-pattern and something we should strictly avoid especially within a pattern library / design system.

[This article by Philip Walton is old](https://css-tricks.com/dangers-stopping-event-propagation/), but still relevant:

> This solution is roughly equivalent to fixing a leaky shower by turning off the water to the bathroom. It works, but it completely ignores the possibility that any other code on the page might need to know about that event.